scripts/ci/errno.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+import re
+import sys
+
+def parse_errno(path):
+    with open(path, 'r') as f:
+        r = re.compile('^\s*#define\s+([A-Z]+)\s+([0-9]+)')
+        errnos = []
+        for line in f:
+            m = r.match(line)
+            #print(f'{line}: {m}')
+            if m:
+                print(f'{m.group(1)}: {m.group(2)}')
+                errnos.append((m.group(1), m.group(2)))
+
+    return errnos
+
+def main():
+
+    minimal = Path("lib/libc/minimal/include/errno.h")
+    newlib = Path("arm-zephyr-eabi/arm-zephyr-eabi/include/sys/errno.h")
+
+    try:
+        minimal = os.environ.get('ZEPHYR_BASE') / minimal
+        newlib = os.environ.get('ZEPHYR_SDK_INSTALL_DIR') / newlib
+    except:
+        print('Environment variable missing')
+        sys.exit(1)
```

Comment:
Better use script arguments to pass required information

Reply:
I am not convinced about this. This is common practice in our GitHub actions, especially for core env vars like those, which are expected to be present. Also makes it much simpler to run locally.

Reply:
As a general practice I think it's bad to exclusively depend on env vars to operate. What I usually do is to use them as defaults if set, something like e.g.
```
    parser.add_argument(
        "--foo",
        default=os.environ.get("FOO")
    )
```